beta/nncf/tensorflow/sparsity/magnitude/operation.py

```diff
@@ -71,8 +75,7 @@ def _create_bkup_weights(layer, w_name):
         return bkup_var
 
     def get_config(self):
-        return {'w_name_to_bkup': self.w_name_to_bkup}
-
-    @classmethod
-    def from_config(cls, config):
-        return cls(config['w_name_to_bkup'])
```

Comment:
Why it was removed?

Reply:
Because method `from_config` of base class`NNCFOperation` is valid for `AsymmetricQuantizer`